Routes/PlaygroundRoutes.js

```diff
@@ -3,10 +3,22 @@ const User = require("../models/User");
 const express = require("express");
 const router = express.Router();
 
-router.get("/api/playground", (req, res) => {
-  Playground.find({})
+router.get("/api/playground/:longitude/:latitude", (req, res) => {
+  const { latitude } = req.params;
+  const { longitude } = req.params;
```

Comment:
you can extract both in one line:
`const { latitued, longitude } = req.params;`